lib/game.rb

```diff
@@ -0,0 +1,167 @@
+require 'pry'
+##
+# => Game handles the functionality of the Tic-Tac-Toe game
```

Comment:
Great bin file above, short length is ideal. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.